xmtp_db/src/encrypted_store/migration_test/update_dm_trigger.rs

```diff
@@ -0,0 +1,38 @@
+use super::*;
+
+#[xmtp_common::test]
+async fn update_dm_trigger() {
+    let db = crate::TestDb::create_database(None).await;
+    db.conn()
+        .raw_query_read(|c| {
+            db.validate(c).unwrap();
+            Ok(())
+        })
+        .unwrap();
+
+    db.conn()
+        .raw_query_write(|conn| {
+            for _ in 0..25 {
+                conn.run_next_migration(MIGRATIONS).unwrap();
+            }
+
+            sql_query(
+                r#"
+            INSERT INTO user_preferences (
+                hmac_key
```

Comment:
This can be deleted now that this migration has been proved and deployed. :smile: